build.gradle

```diff
@@ -39,7 +39,7 @@ allprojects {
         constraintLayoutVersion   =    '2.0.4'
         exifInterfaceVersion      =    '1.3.2'
         duoVersion                =    '1.0.0-alpha01'
-        tokenautocompleteVersion  =    '2.0.8'
+        tokenautocompleteVersion  =    '4.0.0-beta04'
```

Comment:
Cannot use beta version for a non-microsoft library. Can you please update it to a stable GA release version?